Doc/data/refcounts.dat

```diff
@@ -1136,6 +1136,10 @@ PyAIter_Check:PyObject*:o:0:
 PyIter_Next:PyObject*::+1:
 PyIter_Next:PyObject*:o:0:
 
+PyIter_NextIter:int:::
+PyIter_NextIter:PyObject*:iter:0:
+PyIter_NextIter:PyObject**:item:+1:
```

Comment:
Function name has a typo.

Reply:
Thanks, I'll get to it in a day or two. It does not have any practical implications, though, because the tooling has no way to mark up reference count implications for output parameters.

Reply:
Fixed with fe41ad7285259c64317537561fa5c36e84870ddf; thanks!